packages/react-native-bridge/index.js

```diff
@@ -352,79 +342,6 @@ export function showXpostSuggestions() {
 	return RNReactNativeGutenbergBridge.showXpostSuggestions();
 }
 
-/**
- * Request the host app to show the block for editing its mediaFiles collection
- *
- * For example, a mediaFiles collection editor can make special handling of visualization
- * in this regard.
- *
- * @param {Array<Map>} mediaFiles    the mediaFiles attribute of the block, containing data about each media item.
- * @param {string}     blockClientId the clientId of the block.
- */
-export function requestMediaFilesEditorLoad( mediaFiles, blockClientId ) {
```

Comment:
Only used in Story block ([reference](https://github.com/Automattic/jetpack/pull/36151/files#diff-3ec47af4510e547616f9dbcdcd2557a8cd539dcaf938c8b1c1a7d5f42fc696a7L51-L54))